src/allow-controll-on-touch.js

```diff
@@ -0,0 +1,9 @@
+function wikiaJWPlayerAllowControllOnTouchDevices(elementId, playerInstance) {
+	playerInstance.on('playerStart', function () {
+		var unmuteIcon = document.querySelector('.jw-autostart-mute'),
+			jwPlayerClassList = document.getElementById(elementId).classList;
```

Comment:
you can use `playerInstance.getContainer()` instead of `document.getElementById(elementId)`